test/built-ins/Temporal/PlainDateTime/constructor-full.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime
+description: Checking an explicitly constructed instance with all arguments
+features: [Temporal]
+includes: [compareArray.js, temporalHelpers.js]
+---*/
+
+const calendar = Temporal.Calendar.from('iso8601');
+const datetime = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, 30, 123, 456, 789, calendar);
+
+TemporalHelpers.assertPlainDateTime(datetime,
+  1976, 11, 'M11', 18, 15, 23, 30, 123, 456, 789,
+  "check instance (all arguments supplied)",
+  undefined,
+  undefined
+);
+
+assert.sameValue(
+  datetime.calendar,
+  calendar,
+  'calendar supplied in constructor can be extracted and is unchanged'
+);
+
+assert.compareArray(
```

Comment:
Normally I'm a fan of conciseness but I'd recommend 5x `assert.sameValue` here because otherwise the assertion message will not help pinpoint what the failure is, if there is one.

Reply:
done

Reply:
Would it make sense to extend `assertPlainDateTime` with additional (optional) arguments to check this data? Then these tests could be put into a single function call.